dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -325,3 +329,12 @@ def log_in_as(user)
   # Note: Safari webdriver does not support actions API
   @browser.action.key_down(:control).send_keys(key).key_up(:control).perform
 end
+
+And(/^I press keys "([^"]*)" for element "([^"]*)"$/) do |key, selector|
+  if key === ":backspace"
+    key = :backspace
```

Comment:
Probably. I spent some amount of time trying to figure out how to do this with Ruby, but eventually gave up. Will look to see if this does what I need. Thanks.